claim/README.md

```diff
@@ -261,18 +261,23 @@ If you are certain firewall and proxy settings are not the issue, you should con
 GitHub](https://github.com/netdata/netdata/issues/new?labels=bug%2C+needs+triage%2C+ACLK&template=bug_report.md&title=ACLK-available-is-false)
 with details about your system and relevant output from `error.log`.
 
-### Unclaim (remove) an Agent from Netdata Cloud
+### Unclaim and reclaim a node
 
-The best method to remove an Agent from Netdata Cloud is to unclaim it by deleting the `cloud.d/` directory in your
-Netdata library directory.
+To unclaim a node from your Space in Netdata Cloud, delete the `cloud.d/` directory in your Netdata library directory.
```

Comment:
I don't think this will unclaim the agent from the cloud - it will still be listed as unreachable in any room it has been added to.